drivers/usb/device/usb_dc_dw.c

```diff
@@ -25,8 +25,15 @@
 #include "clk.h"
 #endif
 
-#define SYS_LOG_LEVEL CONFIG_SYS_LOG_USB_DRIVER_LEVEL
-#include <logging/sys_log.h>
+#define LOG_LEVEL CONFIG_USB_DRIVER_LOG_LEVEL
+#include <logging/log.h>
+LOG_MODULE_REGISTER(usb_dc_dw);
+
+#define USB_DBG(fmt, ...) LOG_DBG("(%p): %s: " fmt, k_current_get(), \
```

Comment:
why are we redefining this here? is it to get the function/line no only? We should avoid this deviation and introduction of subsystem specific macros and instead look how we can get this functionality from the core logger. @nordic-krch, is there a way for us to include __func__ in the output in a generic way?